### DISCUSSION TITLE
5.6.0

### DISCUSSION THREAD

**Discussion by niden** [phalcon/cphalcon#16505] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/phalcon/cphalcon/releases/tag/v5.6.0'>5.6.0</a>.</em>

**Comment by s-ohnishi** [comment]
`Use of "static" in callables` is still occurring. When will this be resolved?
I want to move to PHP8.2 as soon as possible.

**Comment by lubber-de** [comment]
Windows Builds 8.1, 8.2, 8.3 again missing :(

**Comment by HiralRajgor** [comment]
Phalcon V5.5 Model->save() and doSave() is consuming more memory, and due to this it is giving the 502 Bad Gateway, till phalcon V5.3 it was working fine, even with V5.4, and in alpine linux phalcon pecl package is also not available, so it is upgrading to 5.5 only, please make 5.2 or lower version of phalcon in phalcon pecl alpine linux packages so my production code can work smoothly.

**Comment by dz3n** [comment]
@niden do you have future plans to add resetColumns(), resetFrom(), resetWhere(), resetGroupBy(), resetHaving(), resetOrderBy(), resetLimit(), resetFlags() to the Phalcon\Mvc\Model\Query\Builder as well. It will be really helpful.

**Reply by niden** [reply]
I didn't to be honest but open an issue and I can see what we can add there.
